source/clear-linux/getting-started/bare-metal-install/bare-metal-install.rst

```diff
@@ -3,36 +3,17 @@
 Clear Linux host OS install on bare metal
 #########################################
 
-Verify your system hardware is supported by |CLOSIA|. |CL| can run on all
-Intel® 64bit processors which support UEFI\* and SSE\* v4.1 streaming SIMD\*
-instructions. The following processor families can run |CL|:
+The following are instructions for installing |CL| on bare metal using a
```

Comment:
Consider changing the sentence to: These instructions  will guide you through the installation of |CL| on bare metal using a :ref:\`bootable USB drive\<create-usb\>\`.